dags/esat.py

```diff
@@ -1,14 +1,21 @@
+from pathlib import Path
+
 from airflow import DAG
 from airflow.decorators import task
 from airflow.models import Variable
 
-from dags.common import dbt, default_dag_args
-from dags.common.esat.helpers import get_data_from_sheet, get_variables
-from dags.common.esat.models import build_esat_model, create_tables, insert_data_to_db
+from dags.common import dbt, default_dag_args, gsheet
+from dags.common.esat.models import DB_SCHEMA, EsatBase, create_tables
 
 
 dag_args = default_dag_args() | {"default_args": dbt.get_default_args()}
 
+VARIABLES_FILE = Path("/opt/airflow/dags/common/esat/ESAT_variables_questionnaire2025.csv")
```

Comment:
```suggestion
VARIABLES_FILE = Path(__file__).parent / "common" / "esat" / "ESAT_variables_questionnaire2025.csv"
```
```suggestion
VARIABLES_FILE = Path(__file__).parent.joinpath("common", "esat", "ESAT_variables_questionnaire2025.csv")
```
Le racine n'étant pas contrôlé par le code c'est mieux d'utiliser un chemin relatif.

Les deux formes font la même chose donc c'est une question de goût et de cas d'utilisation.